### PR TITLE
Update to latest vscode-languageclient

### PR DESCRIPTION
**Pull request by stamblerre**
The next version of the vscode-languageclient has been released.

### CODE REVIEW

**Comment by stamblerre** [comment]
There are also a few major version updates there too, and I actually used the https://www.npmjs.com/package/npm-check-updates tool to generate this PR. I find it simplifies the process, but if you'd prefer to only update the single dependency I am happy to revert the other changes.

**Comment by ramya-rao-a** [comment]
Major version updates implies that there can be breaking changes in the product.
Since I don't usually have the time to confirm that none of the breaking changes affect the extension, I avoid updating major versions unless I absolutely have to.

**Comment by stamblerre** [comment]
Sounds good. I edited the PR to only update the vscode-languageclient dependency.

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "vscode-debugadapter": "^1.36.0",
     "vscode-debugprotocol": "^1.36.0",
     "vscode-extension-telemetry": "^0.1.2",
-    "vscode-languageclient": "~5.2.1",
+    "vscode-languageclient": "~6.0.1",
```
Comment (ramya-rao-a) [inline package.json]: Do you want to consider changing the ~ here to ^?

Comment (stamblerre) [reply package.json]: Done. I wasn't sure if there was some reason that was a ~.

Comment (ramya-rao-a) [reply package.json]: There was a time when a new feature got added which wasnt supported yet, caused some problems, so we moved to getting only patch updates.

**Pull request merged**